pacai/student/multiAgents.py

```diff
@@ -87,14 +87,11 @@ def evaluationFunction(self, currentGameState, action):
 class MinimaxAgent(MultiAgentSearchAgent):
     """
     Your minimax agent (question 2)
-    """
 
-    def __init__(self, index):
-        super().__init__(index)
+    Method to Implement:
 
-    def getAction(self, gameState):
-        """
-        Returns the minimax action from the current gameState using self.treeDepth
+    - def getAction(self, gameState):
+        - Returns the minimax action from the current gameState using self.treeDepth
         and self.evaluationFunction.
 
         Here are some method calls that might be useful when implementing minimax.
```

Comment:
Move this portion of the comment above "Methods to implement" and take indent in one